content/docs/command-reference/fetch.md

```diff
@@ -277,4 +277,4 @@ into the workspace (with `dvc repro train.dvc`).
 > Note that in this example project, the last stage file `evaluate.dvc` doesn't
 > add any more data files than those form previous stages, so at this point all
 > of the data for this pipeline is cached and `dvc status -c` would output
-> `Data and pipelines are up to date.`
+> `Cache and remote '{remote}' are in sync.`
```

Comment:
we can potentially use `{name}` instead? instead of `remote {remote}`

Reply:
I changed it to `Cache and remote 'myremote' are in sync.` which is what the message would actually look like with our generic remote name (used a lot in the add/modify examples). In a couple of instances I used `r1` to match the context. See 91c55fef